dora/integration/fuse/src/test/java/alluxio/fuse/AlluxioJnrFuseFileSystemTest.java

```diff
@@ -78,6 +79,7 @@
  */
 @RunWith(PowerMockRunner.class)
 @PrepareForTest({BlockMasterClient.Factory.class})
+@Ignore("we don't maintain jnr any more")
```

Comment:
good catch, we should create a separate PR to remove JNR

Reply:
@LuQQiu is planning to remove all JNR related code, right?